docs/source/relayer.rst

```diff
@@ -1,13 +1,40 @@
 Relayer
 =======
 
+Relayer is a node.js server application, that allows interacting with blockchain without wallet.
```

Comment:
is a RESTFUL JSON RPC API server written in node.js and express.js, that allows interacting with wallet contract using meta-transactions.

Reply:
Missing a connection sentence like: "Below are instructions to run relayer."